pkgs/fc/ceph/src/fc/ceph/maintenance/images_nautilus.py

```diff
@@ -13,134 +15,93 @@
 import tempfile
 import time
 import traceback
+import xmlrpc.client
+from typing import Iterable, Optional
 
 import requests
-from fc.ceph.util import run
-
-RELEASES = [
-    "fc-20.09-production",
-    "fc-21.05-dev",
-    "fc-21.05-staging",
-    "fc-21.05-production",
-    "fc-21.11-dev",
-    "fc-21.11-staging",
-    "fc-21.11-production",
-    "fc-22.05-dev",
-    "fc-22.05-staging",
-    "fc-22.05-production",
-    "fc-22.11-dev",
-    "fc-22.11-staging",
-    "fc-22.11-production",
-    "fc-23.05-dev",
-    "fc-23.05-staging",
-    "fc-23.05-production",
-    "fc-23.11-dev",
-    "fc-23.11-staging",
-    "fc-23.11-production",
-    "fc-24.05-dev",
-    "fc-24.05-staging",
-    "fc-24.05-production",
-    "pre-fc-24.05",
-    "ts-test",
-]
+from fc.ceph.util import directory, run
+
 CEPH_CONF = "/etc/ceph/ceph.conf"
 CEPH_CLIENT = socket.gethostname()
 CEPH_POOL = "rbd.hdd"
 LOCK_COOKIE = "{}.{}".format(CEPH_CLIENT, os.getpid())
 
+DEFAULT_RELEASE_GLOBS = [
+    "fc-*-dev",
+    "fc-*-staging",
+    "fc-*-production",
+    "pre-fc-*",
+    "ts-test",
+]
+
```

Comment:
Lets remove this globbing. If we need filtering, I'd rather delegate that to a feature in the directory at some point.